client/src/App.js

```diff
@@ -6,16 +6,30 @@ import ManagerPage from './pages/ManagerPage'
 import ManagerIndividualPage from './pages/ManagerIndividualPage'
 import ServerDownPage from './pages/ServerDownPage'
 import ErrorPage from './pages/ErrorPage'
+// eslint-disable-next-line no-unused-vars
```

Comment:
We can I guess, I am not completely sure about these lines. Cris used them because ESLint was apperently causing trouble to the cookies. I can try to run the code without these comments and let you know.